packages/atomic/src/pages/index.html

```diff
@@ -347,16 +348,14 @@
                   </atomic-table-element>
                   <atomic-result-section-children>
                     <atomic-result-children>
+                      <!-- CHILD -->
                       <atomic-result-children-template>
+                        <!-- CHILD TEMPLATE -->
                         <template>
                           <atomic-result-link></atomic-result-link>
-                          <atomic-result-children>
-                            <atomic-result-children-template>
-                              <template>
-                                <atomic-result-link></atomic-result-link>
-                              </template>
-                            </atomic-result-children-template>
-                          </atomic-result-children>
+                          <div style="padding-left: 15px">
+                            <atomic-result-children inherit-templates> <!-- GRANDCHILD --> </atomic-result-children>
```

Comment:
Ohh interesting, what does `inherit-templates` do?

Reply:
so, usually if you add a `atomic-result-children` component without any `atomic-result-children-template`, you get a warning (because we don't have default templates for children results, so you need to specify them.
howeeeverrr if you then add `inherit-templates` to it, you are specifying that you want your grandchild result to use the same template as you specified for the child results (so it only works when you have nested `atomic-result-children`)

a few different cases for structure + the template options that the results will have:
```
  atomic-result-children
    template 1
    template 2
    atomic-result-children with inherit-templates

parent 
  child: template 1, template 2
    grandchild: template 1, template 2
```

```
  atomic-result-children
    template 1
    template 2
    atomic-result-children with inherit-templates
      template 3
      template 4

parent 
  child: template 1, template 2
    grandchild: template 3, template 4, template 1, template 2 (in this order)
```

```
  atomic-result-children
    template 1
    template 2
    atomic-result-children WITHOUT inherit-templates
      template 3
      template 4

parent 
  child: template 1, template 2
    grandchild: template 3, template 4
```